### ISSUE TITLE
False positive missing_docs with Swift 2.3

### ISSUE THREAD

**Issue by Loyolny** [realm/SwiftLint#855]
``` bash
MacBook-Pro:Example loyolny$ swiftlint version
0.12.0
```

I'm using Xcode 8 with Swift 2.3. Here's the class:

``` swift
import UIKit

/// Just a sample class
public class SampleClass {

    /**
     Example method

     - returns: Bool (true)
     */
    func exampleMethod() -> Bool {
        // Returns true to satisfy example unit test
        return true
    }

    /**
     Example method

     - returns: Bool (true)
     */
    func exampleMethod2() -> Bool {
        // Returns true to satisfy example unit test
        return true
    }
}
```

When I build the project in Xcode, it passes swxftlint phase without any warnings (other warnings appear, it's just missing_docs problem)

When I run swiftlint in console, that's what I get: `/Users/[...]/Pod/Classes/SampleClass.swift:12:8: warning: Missing Docs Violation: Public declarations should be documented. (missing_docs)`

So I guess console gives me false positive missing_docs, because the class and functions are described.

**Comment by tapi** [comment]
We're seeing similar behaviour except its public properties

```swift
// An object describing a round within a bracket
public class Round {
	/// true if the round is the final round within the bracket, false otherwise.
	public var finals: Bool = false
}
```

For us it looks as though the difference between Xcode and command line is probably because we have swiftlint running in the main app targets build phase but the model is part of another target.

Either way the class and properties are documented and swiftlint is reporting failures

**Comment by Loyolny** [comment]
Well, I had to stop using this rule, because it's made my builds on Bamboo fail :/

**Comment by nhgrif** [comment]
The title of this issue says Swift 2.3, but the original comment says Swift 3.0.  Is it happening for both?

**Comment by tapi** [comment]
We are seeing the issue with Swift 2.3, can't speak to wether or not it affects 3.0

**Comment by Loyolny** [comment]
@nhgrif Oh, man. It was a mistake. It's Swift 2.3

**Comment by jpsim** [comment]
This is a duplicate of #728. Please see my comment there (https://github.com/realm/SwiftLint/issues/728#issuecomment-263009386) for more details and suggestions on how to resolve the situation. Thanks!
